setup.py

```diff
@@ -1,61 +1,3 @@
-# -*- coding: utf-8 -*-
-try:
-    from setuptools import setup, find_packages
-except ImportError:
-    import distribute_setup
+from setuptools import setup
```

Comment:
Why keep this file?

Reply:
It's still expected by some infrastructure, for instance GitHub Dependency graph get's the package name here. It's advised to keep around for now, but I expect these won't be around in later years.

Reply:
Interesting. Can we get a list of tools that are still using it? That way we can track when this should be removed. GitHub Dependency graph seems to support it [now](https://github.blog/changelog/2023-02-13-dependency-graph-supports-the-python-pep-621-standard/).

Reply:
Ok, agree to remove this file.